test_pull_requests

```diff
@@ -152,6 +178,97 @@ def submitted_tests_for_branch(branch)
   $submitted_tests[branch] ? $submitted_tests[branch] : $submitted_tests['*']
 end
 
+# Pulls out as much metadata from the bot comment as it can, including
```

Comment:
If we really care we should be targetting RDoc (https://github.com/rdoc/rdoc). That would be appropriate on a larger project, but on a single-file script I think the extra work doesn't give good value.